rllib/models/tests/test_convtranspose2d_stack.py

```diff
@@ -0,0 +1,53 @@
+import cv2
+import numpy as np
+import unittest
+
+from ray.rllib.models.torch.modules.convtranspose2d_stack import \
+    ConvTranspose2DStack
+from ray.rllib.utils.framework import try_import_torch, try_import_tf
+
+torch, nn = try_import_torch()
+tf1, tf, tfv = try_import_tf()
+
+
+class TestConvTranspose2DStack(unittest.TestCase):
+    """Tests our ConvTranspose2D Stack modules/layers."""
+
+    def test_convtranspose2d_stack(self):
+        """Tests, whether the conv2d stack can be trained to predict an image.
+        """
+        batch_size = 128
+        input_size = 1
+        module = ConvTranspose2DStack(input_size=input_size)
+        optim = torch.optim.Adam(module.parameters(), lr=0.0001)
+        img = cv2.imread("../../tests/data/images/obstacle_tower.png")
+        # Make channels first.
+        img = np.transpose(img, (2, 0, 1))
```

Comment:
Should be a generic pixel preprocessing function

Reply:
+1, do we already have anything like that somewhere?

Reply:
Not sure... If you look in Dreamer dmc2gym in examples/envs, there is similar preprocessing